src/projects/automation/datastores/views.md

```diff
@@ -0,0 +1,89 @@
+```sql
+JOIN Directors ON Films.Director = Directors.Name
+```
+
+But what would happen if the Films table contained a Director that didn't exist in your Directors table? Well, fortunately you can specify what _type_ of join you wish to perform.
+
+There are three types of Join that you can perform in a View, an (inner) join, a left join and a right join. Let's take a look at how each of these types of join behave.
+
+**Inner Join**
+
+An inner join will only include the rows from both table where the data exists in both the base table (the left table) and the external table (right table). In our example above, this means that it will only include rows of data in our View where the Director exists in both the Films table and the Directors table. An inner join can be visualised as the following:
+
+![Inner Join](/src/assets/inner_join.png)
+
+**Left Join**
+
+A Left Join will include all rows in your left table (base table) and where it can it will also include the rows that match the criteria from the right table (external table). In our example, this will include all the rows from the Films table, but only populate the Director information if that Director existed in the Directors table. A Left join can be visualised as the following:
+
+![Left Join](/src/assets/left_join.png)
+
+**Right Join**
+
+A Right Join is the opposite of a Left Join, it will include all rows from the right table (external table) and only include data from the left table (base table) if it met the join criteria. A right join would behave very similarly to an inner join in our example of Films and Directors but it would also include all the Director information even if there weren't any corresponding Films. A right join can be visualised as the following:
+
+![Right Join](/src/assets/right_join.png)
+
+#### Queries
+
+You would normally use a View because you want to frequently reference a subset of data from one or more tables and a Query is how we would define what that subset contains. Imagine, for example, you have a table of films which includes the name, description, director, genre and year of release. You might want to create a View for each genre so you would construct a View with a query such as:
+
+```sql
+WHERE genre = "Action"
+OR genre = "Adventure"
+```
+
+this would create a View which would display Action and Adventure films.
+
+You can perform a query on any of the columns from any of the tables you are using in your View, so this includes querying any tables you are using in joins.
+
+#### Aliases
+
+Whilst Joins are great for displaying related data it can lead to duplicate column names as you might be adding a Created date column from your base table and a Created date from your external table. This could lead to some serious confusion as you then end up with a View which has two columns named the same! That's where Aliases come in to save the day. An alias allows you to give each column in your View a unique name so you can avoid this confusion.
```

Comment:
`That's where Aliases come in to save the day`? Surely either `That's where Aliases come in` OR `Aliases come in to save the day` (preferably the former)